src/main/java/duke/main/Ui.java

```diff
@@ -0,0 +1,45 @@
+package duke.main;
+
+import duke.utils.Constants;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    private String response;
+    private boolean isExit;
+    Scanner sc = new Scanner(System.in);
+
+    public Ui() {
+        isExit = false;
+    }
+
+    public void setResponse(String response) {
+        this.response = response;
+    }
+
```

Comment:
We usually print out certain same outputs frequently, it is better to create a new method for it. For example, responseGoodbye, or responseError, because the message is always the same. Instead of calling setResponse("dummy message") in one place, then calling setResponse("dummy message") again in another place. It is better to just call printDummyMessage().